lib/utils.js

```diff
@@ -280,9 +301,9 @@ const isChangelogLabel = (label) => {
  * @param {string} type
  */
 const getLastReviewOfSpecificType = async (
-  context,
-  type,
-  pullRequestNumber
+    context,
+    type,
+    pullRequestNumber
```

Comment:
Why this change?

Reply:
It's due to the eslint errors I was getting locally.